test/main.fmf

```diff
@@ -0,0 +1,25 @@
+# Only common dependencies installable from default repos are mentioned here.
+# All other dependencies are handled in podman-tests.sh
+require:
+    - make
+    - policycoreutils
+
+/basic_check:
+    summary: Run basic checks
+    test: make basic_check
```

Comment:
```suggestion
require:
    - make
    - policycoreutils

test: "make "

/basic_check:
    summary: Run basic checks
    test+: basic_check
```

You can do stuff like these. Not sure if it makes sense here or not. Another design I had was to use `test: make $TMT_TEST_NAME`, but that might need some refactoring. Personally though I tend to stay away from Makefile, but that's a personal taste.

Reply:
main reason to use Makefile was for a human to quickly run some make command, without having to bother with TMT. For that same reason, I guess it's best to have the package dependencies / copr enablement conditionals within the script itself along with some warnings whenever a copr or epel is being enabled.

But I do hear your suggestions, thanks for those. I'll probably switch to idiomatic packit if everyone is comfortable enough with TMT.

Reply:
Cool, however the team feels more comfortable. The trade here is mostly having to maintain the installation and logging manually, vs delegating to tmt.